README.md

```
@@ -1,2 +1,416 @@
+# AI 기반 고객 이탈 예측 및 리텐션 ROI 최적화 시스템
+
+> 이커머스 고객 이탈 예측 + Uplift Modeling 기반 리텐션 ROI 최적화 End-to-End 시스템
+
+[![Python](https://img.shields.io/badge/Python-3.10+-blue.svg)](https://www.python.org/)
+[![PyTorch](https://img.shields.io/badge/PyTorch-2.0+-red.svg)](https://pytorch.org/)
+[![Docker](https://img.shields.io/badge/Docker-Compose-blue.svg)](https://docs.docker.com/compose/)
+[![Status](https://img.shields.io/badge/Status-In%20Development-yellow.svg)]()
+
+## 목차
+
+1. [프로젝트 소개](#-프로젝트-소개)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)